### PR TITLE
Update FAQ page

### PR DESCRIPTION
**Pull request by joeyballentine**
image

### CODE REVIEW

**Comment by Kim2091** [comment]
This is very well written, it's pretty much perfect. The only suggestion I have is to add an example or reference for what a "supported program" is, such as:

> "Model" is a generic term referring to a Machine Leaming Model. On this site, it refers to the file you would download and use inside a supported program (Refer to Maintained Programs below for examples). Elsewhere, it can refer to the model's architecture, the weights, or all of the above.

**Review by RunDevelopment — APPROVED** [review]

File: docs/faq.md [hunk 1]
```diff
@@ -1,15 +1,63 @@
...
+  - [Wiki]() | [GitHub](https://github.com/chaiNNer-org/chaiNNer) | [Website](https://chainner.app/)
+  - chaiNNer is a node-based image processing GUI that can also be used for image upscaling. It has the most support for models listed on OpenModelDB and can be run with PyTorch (CUDA), ONNX (CUDA), and NCNN (AMD/Intel). Being node based, it allows you to process your images with a lot more control, at the cost of being a bit more complex. Besides upscaling, chaiNNer also has a variety of other use cases and is a very versatile program.
+
+- **enhancr** (open-source | paid)
+  - [Wiki]() | [GitHub](https://github.com/mafiosnik777/enhancr)
+  - Meant for fast video frame upscaling and interpolation, enhancr takes advantage of Nvidia's TensorRT to provide fast processing on video. It supports a variety of models, including many of the custom ESRGAN ones from OpenModelDB.
+
+- **VSGAN-TensorRT-docker** (open-source | free)
+  - [Wiki]() | [GitHub](https://github.com/styler00dollar/VSGAN-tensorrt-docker)
+  - A more complicated but free alternative for fast video upscaling with TensorRT. Supports ann insane amount of models, but requires manual parameter selection.
```
[the remark this answers is not in the document]
Comment (joeyballentine) [reply docs/faq.md:35]: My damn keyboard....

**Comment by RunDevelopment** [comment]
I'm also confused as to what the "Wiki |" means.

**Comment by joeyballentine** [comment]
> I'm also confused as to what the "Wiki |" means.

Placeholders for links to the individual pages for each program that i haven't made yet

**Pull request merged**